WalletWasabi.Tests/RegTests.cs

```diff
@@ -3101,7 +3101,7 @@ public async Task CcjFeeTestsAsync()
 
 					if (chaumianClient != null)
 					{
-						await chaumianClient.DequeueAllCoinsFromMixAsync();
+						await chaumianClient.DequeueAllCoinsFromMixAsync("No reason given");
```

Comment:
Shouldn't it be empty string for consistency?

```suggestion
						await chaumianClient.DequeueAllCoinsFromMixAsync("");
```